extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -233,6 +290,14 @@ oppia.directive('imageWithRegionsEditor', [
           }
           $scope.userIsCurrentlyDrawing = false;
           $scope.userIsCurrentlyDragging = false;
+          $scope.userIsCurrentlyResizing = false;
+          if ($scope.outOfRegion) {
+            $scope.xDirection = 0;
+            $scope.yDirection = 0;
+          }
```

Comment:
But it seems that they are always set to 0 if I drag from north to south (for example) as you mentioned. In that case, shouldn't I still be able to resize from the same position I left off like you said?